internal/infrastructure/mongo/mongodoc/scene_alignsystem.go

```diff
@@ -0,0 +1,129 @@
+package mongodoc
+
+import (
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/reearth/reearth-backend/pkg/scene"
+)
+
+type WidgetLocation struct {
+	Zone    string
+	Section string
+	Area    string
+}
+
+type WidgetLayout struct {
+	Extendable      bool
+	Extended        bool
+	Floating        bool
+	DefaultLocation *WidgetLocation
+}
+
+type WidgetArea struct {
+	WidgetIds []string
+	Align     string
+}
+
+type WidgetSection struct {
+	Top    WidgetArea
+	Middle WidgetArea
+	Bottom WidgetArea
+}
+
+type WidgetZone struct {
+	Left   WidgetSection
+	Center WidgetSection
+	Right  WidgetSection
+}
+
+type SceneAlignSystemDocument struct {
+	Inner WidgetZone
+	Outer WidgetZone
+}
+
+func NewWidgetAlignSystem(was scene.WidgetAlignSystem) *SceneAlignSystemDocument {
```

Comment:
```suggestion
func NewWidgetAlignSystem(was *scene.WidgetAlignSystem) *SceneAlignSystemDocument {
```

Since the domain model assumes that the WidgetAlignSystem is handled by reference, this New method should also accept the WidgetAlignSystem by reference. The same goes for the others.